pkg/kubelet/kubelet_test.go

```diff
@@ -2410,9 +2410,13 @@ func TestSyncLabels(t *testing.T) {
 						}
 						for _, action := range actions {
 							if action.GetVerb() == "patch" {
-								patchAction := action.(core.PatchActionImpl)
-								var err error
-								savedNode, err = applyNodeStatusPatch(test.existingNode, patchAction.GetPatch())
+								var (
+									err          error
+									patchAction  = action.(core.PatchActionImpl)
+									patchContent = patchAction.GetPatch()
+								)
+								t.Logf("using patch %s to applyNodeStatusPatch", patchContent)
```

Comment:
No need of this line. The main problem is with node object creation that can be racy.

Reply:
It can help for later debug. Since this pr has been go through two approve process, it currently only needs a millstone label to merge. So better keep it.😁

Reply:
Done